### PR TITLE
Add codex package for CNV analysis using exome sequencing data

### PR DESCRIPTION
**Pull request by drramki-chop**

### CODE REVIEW

File: recipes/bioconductor-codex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package:
+  name: bioconductor-codex
+  version: 1.2.0
+
+source:
+  fn: CODEX_1.2.0.tar.gz
+  url: https://www.bioconductor.org/packages/release/bioc/src/contrib/CODEX_1.2.0.tar.gz
```
Comment (johanneskoester) [inline recipes/bioconductor-codex/meta.yaml]: Can you add a bioarchive url as well? The bioconductor URLs are not persistent unfortunately.

Comment (johanneskoester) [inline recipes/bioconductor-codex/meta.yaml]: Did you use the bioconductor script in the scripts directory? I thought it would do that automatically...

**Comment by drramki-chop** [comment]
Any reason why this didn't go through the checks?

**Comment by johanneskoester** [comment]
Sometimes travis does not receive the hook properly. I guess you have to edit the code a bit in order to trigger a new check. E.g. by adding a sustainable URL from bioarchive.

**Pull request closed without merging**